app/views/api/v1/accounts/create.json.jbuilder

```diff
@@ -6,6 +6,7 @@ json.data do
   json.display_name resource.display_name
   json.email resource.email
   json.account_id @account.id
+  json.created_at resource.created_at
```

Comment:
This could be the reason for https://linear.app/chatwoot/issue/CW-1315/the-contacts-are-not-synchronized-to-chatwoot-contacts